NeuroID/build.gradle

```diff
@@ -124,6 +149,8 @@ dependencies {
     implementation 'com.squareup.retrofit2:retrofit:2.9.0'
     implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
     implementation 'com.squareup.okhttp3:okhttp:4.9.2'
+    implementation "androidx.navigation:navigation-compose:2.5.3"
+    implementation "androidx.compose.ui:ui:1.2.0"
```

Comment:
these 2 dependencies are duplicates. Prob a bad merge b/w Kurt and my PR. Sorry!

Reply:
working on this. commit coming soon.